pkg/controller/user/index.go

```diff
@@ -36,8 +36,26 @@ func (c *Controller) HandleIndex() http.Handler {
 			return
 		}
 
+		creationCounts1d := make(map[uint]uint64)
```

Comment:
This is identical to the other ones? Maybe we should make a helper or separate package?

Reply:
opened issue to track this since merged before resolved fully: https://github.com/google/exposure-notifications-verification-server/issues/199